documentation/asciidoc/topics/attributes/community-attributes.adoc

```diff
@@ -23,6 +23,7 @@
 :backup_path: /opt/infinispan/backups
 :server_image: quay.io/infinispan/server
 :server_image_version: 14.0.1
+:operand_version: 14.0.1
```

Comment:
Why do we need this as well as `server_image_version`?

Reply:
Hi @ryanemerson , the versions are the same in community but not in the product. There we need 8.4.0-1. I understand it's another version field to maintain but that's the solution I came up with for now.